cmd/dfget/app/gen_doc.go

```diff
@@ -0,0 +1,39 @@
+package app
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+// GenDocCommand is used to implement 'gen-doc' command.
+type GenDocCommand struct {
+	cmd *cobra.Command
+}
+
+func init() {
+	genDocCommand := &GenDocCommand{}
+	genDocCommand.cmd = &cobra.Command{
+		Use:           "gen-doc",
+		Short:         "Generate Document for dfget command line tool with MarkDown format",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return genDocCommand.runGenDoc(args)
+		},
+	}
+	rootCmd.AddCommand(genDocCommand.cmd)
+}
+
+func (g *GenDocCommand) runGenDoc(args []string) error {
```

Comment:
Could you please implement the related part of `dfdaemon` by he way?

Reply:
Yes, I will do that in a separate PR.